cmd/commands/cli/command_cli.go

```diff
@@ -392,5 +405,7 @@ func newAutocompleter(tequilapi *tequilapi_client.Client, proposals []tequilapi_
 				getIdentityOptionList(tequilapi),
 			),
 		),
+		readline.PcItem("warranty"),
```

Comment:
What about licence prefix here, to be consistent with other places

Reply:
Added nesting, which makes more sense in CLI.